src/global-firestore-data.ts

```diff
@@ -91,8 +91,11 @@ export const chooseSelectableRequirementOption = (
     .set(selectableRequirementChoices);
 };
 
-const getSubjectColor = (subjectColors: Record<string, string>, subject: string): string => {
-  if (subjectColors[subject]) return subjectColors[subject];
+const getSubjectColor = (
+  subjectColors: Record<string, string>,
+  subject: string
+): readonly [string, boolean] => {
+  if (subjectColors[subject]) return [subjectColors[subject], false];
```

Comment:
Nit: Can you write a // comment that explains what the boolean means? I know that it indicates whether or not the color was changed, but a comment in the code might be clearer for other devs. Your call.